README.md

```diff
@@ -1,5 +1,5 @@
-## Awesome Books App
-A To Do List app built with Javascript and Webpack
```

Comment:
- [ ] Please change the header of your README file as it refers to an older project.